src/Auth/BaseAuthenticate.php

```diff
@@ -37,6 +37,7 @@
      * - `userModel` The alias for users table, defaults to Users.
      * - `scope` Additional conditions to use when looking up and authenticating users,
      *    i.e. `['Users.is_active' => 1].`
+     * - `finder` The finder method to use to fetch user record. Defaults to 'all'.
```

Comment:
We could probably deprecate the `contain` & `scope` options. The new `finder` option gives people more control/flexibility than those other options.